### DISCUSSION TITLE
argo-workflows/fields/

### DISCUSSION THREAD

**Discussion by giscus** [argoproj/argo-workflows#9494] [General]
# argo-workflows/fields/

https://argoproj.github.io/argo-workflows/fields/

**Comment by wenijinew** [comment]
This is one dead link?
https://git.io.k8s.community/contributors/devel/sig-architecture/api-conventions.md#resources

**Reply by wenijinew** [reply]
I guess it should be https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#resources now.

**Comment by driv** [comment]
If I set both `secondsAfterSuccess` and `secondsAfterFailure`, will setting `secondsAfterCompletion` change anything?

**Comment by metasim** [comment]
Where's the field reference for `WorkflowEventBinding`?
